enzy_htp/_interface/amber_interface.py

```diff
@@ -336,11 +338,16 @@ def _parameterize_modified_res(self, maa: ModifiedResidue, gaff_type: str) -> Tu
         else:
             # 1. generate ac if not found
             mol_desc_path = f"{self.ncaa_param_lib_path}/{maa.name}_{target_method}.ac" # the search ensured no existing file named this
-            self.parent_interface.antechamber_ncaa_to_moldesc(ncaa=maa,
+            self.parent_interface.antechamber_ncaa_to_moldesc(ncaa=maa_region,
                                                               out_path=mol_desc_path,
                                                               gaff_type=gaff_type)
         # 2.1 fix the wrong atom type given by antechamber
 
+
+        # create mc file
+        mc_path = f"{self.ncaa_param_lib_path}/{maa.name}_{target_method}.mc" # the search ensured no existing file named this
+        self.parent_interface.make_mc_file(maa_region=maa_region, out_path=mc_path)
+
         # 3. run prepgen on ac & mc get prepin
         prepin_path = fs.get_valid_temp_name(
             f"{self.ncaa_param_lib_path}/{maa.name}.prepin")
```

Comment:
Let's also finish all the following steps in this PR so that we know if our .mc and .ac are legit